README.md

```diff
@@ -4,8 +4,23 @@ Server capable of storing data in a database and CRUDing it, developed as part o
 
 ## Environment and tools
 
-TODO
+- Node
+- Typescript
+- GraphQL
+- ApolloServer
+- TypeORM
+- Docker
 
 ## Running and debugging
 
-TODO
\ No newline at end of file
+1. Make sure the node version from `.nvmrc` is being used.
+
+2. Create and start local databases. This requires a [Docker environment](https://docs.docker.com/compose/install/) set up.
+```
+docker-compose up -d
+```
+
+3. Execute the server: 
```

Comment:
talvez o `npm install` antes?

dica sobre arquivos .md: você pode deixar tudo com 1. que na hora de visualizar incrementa sozinho,
```
1. Um
1. Dois
1. Três
```
vira
1. Um
1. Dois
1. Três

e ai fica mais fácil de adicionar etapas intermediárias sem quebrar tudo haha

Reply:
normalmente eu coloco desse jeito mesmo, mas nesse caso não está incrementando sozinho. Por causa do bloco de código, talvez? 🤔

Reply:
aaah,  verdade, tem que dar uma indentada no bloco inteiro se for o caso, mas ai mó trampo 😄 
1. Um
   
     ```
    console.log('Hello world');
     ```
1. Dois